app/helpers/navigation_helper.rb

```diff
@@ -34,10 +34,88 @@ def local_assigned_boolean(local, default)
     local == "true"
   end
 
+  def family_info_progress_hash
+    if @change_plan.present?
+      {step: '1', option_hash: qle_nav_options}
+    elsif @type == "employee"
+      {step: '4', option_hash: sep_nav_options}
+    else
+      {step: '3', option_hash: individual_nav_options}
+    end
+  end
+
+  def plan_shopping_progress_hash
+    if @change_plan.blank? && @market_kind == "individual"
+      if @enrollment_kind.blank? && is_under_open_enrollment?
+        {step: '3', option_hash: individual_nav_options}
+      else
+        {step: '4', option_hash: sep_nav_options}
+      end
+    elsif @change_plan == "change_by_qle"
+      {step: '1', option_hash: qle_nav_options}
+    elsif @change_plan == "change_plan"
+      if (@market_kind == "individual" && !is_under_open_enrollment?) || @enrollment_kind == 'sep'
+        {step: '2', option_hash: shop_for_plans_nav_options}
+      else
+        {step: '1', option_hash: sep_shop_for_plans_nav_options}
+      end
+    end
+  end
+
   def sign_up_nav_options
```

Comment:
nitpick - can we move this def above the progress_hash helpers? maybe even add comments marking which of these defs belong to the signup flow and which of these defs belong to the coverage experiences? just a little jarring to me to have sign_up_nav_options be between a bunch of defs for a separate experience...